package.json

```diff
@@ -24,9 +24,7 @@
   },
   "homepage": "https://github.com/FormidableLabs/builder-docs#readme",
   "devDependencies": {
-    "babel-core": "6.3.26",
-    "babel-loader": "6.2.1",
-    "builder": "https://github.com/FormidableLabs/builder",
+    "builder": "2.5.0",
```

Comment:
Keep this pegged to `"builder": "FormidableLabs/builder",` so we get any updates the master README.